ntbs-service/Startup.cs

```diff
@@ -55,6 +57,9 @@ public void ConfigureServices(IServiceCollection services)
                     {
                         options.MetadataAddress = adfsConfig["AdfsUrl"] + "/FederationMetadata/2007-06/FederationMetadata.xml";
                         options.Wtrealm = adfsConfig["Wtrealm"];
+                        options.BackchannelHttpHandler = new HttpClientHandler() {
```

Comment:
Can we leave a TODO tagged against the ssl ticket? just so its super clear we're not intending to leave this in long-term at all!